automated-comments/go/general/syntax_error.md

```diff
@@ -0,0 +1 @@
+Unfortunately there is a syntax error in this solution. Try running the tests in a terminal with `go test`.
```

Comment:
```suggestion
Unfortunately there is a syntax error in this solution. Run `go test` in a terminal.
```

Reply:
To me, it looks a little bit like the student might think that running `go test` might solve the syntax error :D

Maybe something like:

```suggestion
Unfortunately there is a syntax error in this solution. This causes the tests to fail, which can be verified by running the tests in a terminal with `go test`.
```

I'm not sure how to elegantly word the action a student should take (fix the syntax error).

Reply:
Maybe in this casy it is obvious ;)